.github/workflows/main.yml

```diff
@@ -10,6 +10,14 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    services:
+      db:
+        image: postgres:11.6-alpine
+        env:
+          POSTGRES_PASSWORD: postgres
```

Comment:
I know is a testing env, but could we handle these params as secrets?
`${{ secrets.POSTGRES_PASSWORD }}` and so ...

Reply:
Yeah it is a good idea, I'm not sure exactly how to accomplish this as we need to edit the `.env.test` file to have a key matching the ENV var without explicitly declaring it

Reply:
I think @sebalopez has a way of doing this, with secrets in the repo?

Reply:
@martinjaimem you can define them here: https://github.com/rootstrap/express-api-base/settings/secrets
and use them from the .yml in the way I mentioned above. In this repo you can find an example https://github.com/rootstrap/github-action-for-trello

Reply:
also if you need the value in an .env file you could add a step in the workflow to write it:
```
echo POSTGRES_PASSWORD=${{ secrets.POSTGRES_PASSWORD }} >> .env.test
```